open_seq2seq/models/text2speech_centaur.py

```diff
@@ -0,0 +1,360 @@
+# Copyright (c) 2018 NVIDIA Corporation
```

Comment:
Same comment as Centaur Loss, can we merge with Text2Speech?

Reply:
Merged.

Reply:
Merged.